server.go

```diff
@@ -2295,9 +2294,9 @@ func (s *server) Stop() error {
 		if err := s.sweeper.Stop(); err != nil {
 			srvrLog.Warnf("failed to stop sweeper: %v", err)
 		}
-
-		s.txPublisher.Stop()
-
+		if err := s.txPublisher.Stop(); err != nil {
```

Comment:
@yyforyongyu not sure but I forgot why the txpublisher had no error return ? Added one now hmm